python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_decorator.py

```diff
@@ -258,59 +257,6 @@ def my_dbt_assets(): ...
             assert output_def.io_manager_key == DEFAULT_IO_MANAGER_KEY
 
 
-@pytest.mark.parametrize(
-    ["partitions_def", "backfill_policy", "expected_backfill_policy"],
-    [
-        (
-            DailyPartitionsDefinition(start_date="2023-01-01"),
-            BackfillPolicy.multi_run(),
-            BackfillPolicy.multi_run(),
-        ),
-        (
-            DailyPartitionsDefinition(start_date="2023-01-01"),
-            None,
-            BackfillPolicy.single_run(),
-        ),
-        (
-            StaticPartitionsDefinition(partition_keys=["A", "B"]),
-            None,
-            None,
-        ),
-        (
-            StaticPartitionsDefinition(partition_keys=["A", "B"]),
-            BackfillPolicy.single_run(),
-            BackfillPolicy.single_run(),
-        ),
-    ],
-    ids=[
-        "use explicit backfill policy for time window",
-        "time window defaults to single run",
-        "non time window has no default backfill policy",
-        "non time window backfill policy is respected",
-    ],
-)
-def test_backfill_policy(
-    test_jaffle_shop_manifest: Dict[str, Any],
-    partitions_def: PartitionsDefinition,
-    backfill_policy: BackfillPolicy,
-    expected_backfill_policy: BackfillPolicy,
-) -> None:
-    class CustomDagsterDbtTranslator(DagsterDbtTranslator):
-        def get_freshness_policy(self, _: Mapping[str, Any]) -> Optional[FreshnessPolicy]:
-            # Disable freshness policies when using static partitions
-            return None
-
-    @dbt_assets(
-        manifest=test_jaffle_shop_manifest,
-        partitions_def=partitions_def,
-        backfill_policy=backfill_policy,
-        dagster_dbt_translator=CustomDagsterDbtTranslator(),
-    )
-    def my_dbt_assets(): ...
-
-    assert my_dbt_assets.backfill_policy == expected_backfill_policy
-
-
```

Comment:
This whole test is no longer needed as there is no special backfill policy logic in `@dbt_assets` anymore.